### ISSUE TITLE
Tag a new release

### ISSUE THREAD

**Issue by mtibben** [lindenb/makefile2graph#40]
Could you tag a new stable release so that distributions like [homebrew](https://formulae.brew.sh/formula/makefile2graph) can pick up the new features recently added?

If possible can you use [Semantic Versioning](https://semver.org/) and update the [`M2G_VERSION`](https://github.com/lindenb/makefile2graph/blob/master/make2graph.c#L46)

**Comment by merkys** [comment]
Switching from CalVer to SemVer is problematic as some distribution management software will not understand that `1.7.0` is newer than `2021.11.06`, take Debian for example where all version comparisons are done in SemVer manner.

**Comment by mtibben** [comment]
calver seems to have been [introduced in 2018](https://github.com/lindenb/makefile2graph/tags) however the binary continues to self-report the version as 1.5 (see [M2G_VERSION](https://github.com/lindenb/makefile2graph/blob/master/make2graph.c#L46))

> Switching from CalVer to SemVer is problematic as some distribution management software will not understand that `1.7.0` is newer than `2021.11.06`, take Debian for example where all version comparisons are done in SemVer manner.

Maybe then it's just roll forward with calver? In any case, a new release would be great

**Comment by mtibben** [comment]
@lindenb if you want me to help out cutting a release feel free to give me appropriate access and I will action

**Comment by 0x5c** [comment]
> Switching from CalVer to SemVer is problematic as some distribution management software will not understand that `1.7.0` is newer than `2021.11.06`, take Debian for example where all version comparisons are done in SemVer manner.

This wouldn't be the first project to do such a versioning scheme change; distros already have tools/mechanisms to handle that.

**Comment by merkys** [comment]
> This wouldn't be the first project to do such a versioning scheme change; distros already have tools/mechanisms to handle that.

You are right, this is doable, although it involves some additional work on distro side.
